generators/client/index.js

```diff
@@ -220,9 +219,9 @@ module.exports = JhipsterClientGenerator.extend({
         configureGlobal: function () {
             // Application name modified, using each technology's conventions
             this.angularAppName = this.getAngularAppName();
-            this.camelizedBaseName = _s.camelize(this.baseName);
-            this.capitalizedBaseName = _s.capitalize(this.baseName);
-            this.dasherizedBaseName = _s.dasherize(_s.camelize(this.baseName,true));
+            this.camelizedBaseName = _.camelCase(this.baseName);
+            this.capitalizedBaseName = _.capitalize(this.baseName);
+            this.dasherizedBaseName = _.kebabCase(_.camelCase(this.baseName,true));
```

Comment:
I think no need to camelCase this can be just `_.kebabCase(this.baseName);` I already did this in master